server/src/test/java/com/saralein/server/response/ResponseTest.java

```diff
@@ -2,26 +2,19 @@
 
 import com.saralein.server.exchange.Cookie;
 import com.saralein.server.exchange.Header;
-import org.junit.Before;
 import org.junit.Test;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
 
 public class ResponseTest {
-    private Response response;
-    private byte[] body;
-
-    @Before
-    public void setUp() {
-        Header header = new Header();
-        header.status(200);
-        header.addHeader("Content-Type", "text/html");
-
-        body = "Hello".getBytes();
-
-        response = new Response(header, body);
+    private boolean cookiesAreEqual(List<Cookie> cookies, List<Cookie> expected) {
+        cookies.sort(Cookie::compareTo);
+        expected.sort(Cookie::compareTo);
+        return cookies.equals(expected);
```

Comment:
This function exists in at least 4 tests. Might want to extract it out. (See comment above)